change/@fluentui-web-components-e7330a7e-73e6-4efd-a09f-84f488b8f917.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 1 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27506&commitId=230f0f25e60f5196d0484af13ad035e791125ff3&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1929 screenshots unchanged, 0 screenshots with different dimensions and 1 screenshots with visible difference.


  <details><summary><b>unknown</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.size+inactive+badge.normal.chromium.png|2| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>